app/models/student_loans_data_importer.rb

```diff
@@ -18,27 +18,25 @@ class StudentLoansDataImporter < CsvImporter::Base
   private
 
   def skip_row_conditions?(row)
-    row.fetch("NINO").blank?
+    row.fetch("NINO").blank? || cast_as_date(row.fetch("Date of birth")).nil?
   end
 
   def row_to_hash(row)
     {
       claim_reference: row.fetch("Claim reference"),
       nino: row.fetch("NINO"),
       full_name: row.fetch("Full name"),
-      date_of_birth: date_of_birth(row.fetch("Date of birth")),
+      date_of_birth: cast_as_date(row.fetch("Date of birth")),
       policy_name: row.fetch("Policy name"),
       no_of_plans_currently_repaying: row.fetch("No of Plans Currently Repaying"),
       plan_type_of_deduction: row.fetch("Plan Type of Deduction"),
       amount: row.fetch("Amount")
     }
   end
 
-  def date_of_birth(dob_str)
-    return unless dob_str.present?
-
-    Date.strptime(dob_str, "%m/%d/%Y")
-  rescue Date::Error
+  def cast_as_date(string)
+    Date.strptime(string, I18n.t("date.formats.day_month_year"))
```

Comment:
I think this is OK as it's unlikely we'll change, but strictly speaking if we are dealing with data from an external source we should specify their formatting rather than assuming it matches ours.